### ISSUE TITLE
is there a fourth tiebreaker?  

### ISSUE THREAD

**Issue by alexmitrani** [climu/openstudyroom#373]
in the fechgo handicap league there are two games.  smontiel and kamelotusky both played me and both beat me.  therefore they both have the same score, number of games, SOS and SODOS.  However, the league table shows kamelotusky first and smontiel second.  is there a fourth tiebreaker?  
This isn't urgent, I'm just curious.  

thanks

![Screenshot_20190402-225025](https://user-images.githubusercontent.com/27974470/55447387-edd49500-5599-11e9-9977-e01a37be0a73.png)

**Comment by climu** [comment]
No. I wouldn't be surprised if it were alphabetical or even random.

Best would be to have two 1st place in that scenario...
